packages/runtime-core/src/renderer.ts

```diff
@@ -794,7 +809,7 @@ function baseCreateRenderer(
     n2: VNode,
     parentComponent: ComponentInternalInstance | null,
     parentSuspense: SuspenseBoundary | null,
-    isSVG: boolean,
+    namespace: 'svg' | 'mathml' | undefined,
```

Comment:
Maybe we can extract the type of namespace instead of repeating code.

Reply:
Sure! Where should I put it? It's used in quite a lot of files.